dataStore/backend-redis.js

```diff
@@ -109,10 +109,19 @@ function configure(opts) {
   });
 }
 
+function flush() {
+  return new Promise(resolve => {
+    client.flushdb(() => {
```

Comment:
Don't you need to check for the error? At least, to not silence it. If you do, you could _promisify_ this method.